gatsby-config.js

```diff
@@ -164,6 +164,16 @@ module.exports = {
                 //   ],
             },
         },
+        {
+            resolve: `gatsby-plugin-offline`,
+            options: {
+              precachePages: [`/`, `/markets/*`, `/trade-types/*`],
```

Comment:
These pages are for testing purposes?

Reply:
Yeah, there's a lot of pages, Planning to start pre-caching these and check it breaks nothing including percy. Gradually, we will increase the number of pre-cached pages.